source/manual/local-frontend-development.html.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Local frontend development
+parent: "/manual.html"
+layout: manual_layout
+section: Frontend
+type: learn
+owner_slack: "#govuk-2ndline"
+last_reviewed_on: 2019-07-08
+review_in: 6 months
+---
+
+Make the following changes to your local setup:
+
+In [govuk_app_config], add your local domain to the allowed CORS domains:
+
+```
+  GOVUK_DOMAINS = [
+    '*.publishing.service.gov.uk',
+    # "*.#{ENV['GOVUK_APP_DOMAIN_EXTERNAL'] || ENV['GOVUK_APP_DOMAIN'] || 'dev.gov.uk'}"
+     '*.dev.gov.uk'
+  ].uniq.freeze
+```
+
+You need to change the frontend app you are working on to use the modified [govuk_app_config]. For example in [frontend] update the Gemfile:
+
+```ruyby
+...
+gem 'govuk_app_config', path: '/var/govuk/govuk_app_config'
+...
+gem 'govuk_publishing_components', path: '/var/govuk/govuk_publishing_components'
+...
+```
+
+and set in `development.rb`:
+
+```ruby
+...
+config.assets.debug = false
+...
+```
+
+Update the Gemfile in [static] to point to [govuk_publishing_components] gem:
+
+```ruby
+...
+gem 'govuk_publishing_components', path: '/var/govuk/govuk_publishing_components'
```

Comment:
Did you manage to get this working? I seem to remember we tried this and it only worked with a relative path, e.g: `../govuk_publishing_components`